src/Sarif.Multitool.Library/ApplyPolicyOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("apply-policy", HelpText = "Apply policies from SARIF log.")]
+    public class ApplyPolicyOptions : SingleFileOptionsBase
+    {
+    }
```

Comment:
>} [](start = 4, length = 1)

Will there in future be an option to specify one out of multiple policies present in the file?

Reply:
It will apply all policies. Should we rename it?

---
In reply to: [506563829](https://github.com/microsoft/sarif-sdk/pull/2118#discussion_r506563829) [](ancestors = 506563829)

Reply:
No need to rename. But... ok, this is a "requirements" question that you should discuss with @michaelcfanning. As the code stands, you apply policies in order, and if two policies refer to the same rule, the last one wins (as your test verifies). Is Michael sure that's what we want?

---
In reply to: [506565036](https://github.com/microsoft/sarif-sdk/pull/2118#discussion_r506565036) [](ancestors = 506565036,506563829)

Reply:
Definitely what we specified for now. 

---
In reply to: [506571247](https://github.com/microsoft/sarif-sdk/pull/2118#discussion_r506571247) [](ancestors = 506571247,506565036,506563829)